### DISCUSSION TITLE
Beautiful, snappy, smooth... PERFECT!

### DISCUSSION THREAD

**Discussion by a0g83agbc84** [CappielloAntonio/tempo#162] [General]
Yes, there are some things that could be improved, but...

Just dropping my most sincere thanks for such an awesome piece of software.

Thanks you! ❤️

**Comment by Neurology0443** [comment]
I was going to post the same ! 

Thank you for such fantastic android app working with subsonic server ! Keep up the good work but also do not forget to take time for yourself.

Your app is gaining traction and you will be overwhelmed by issues, bugs and other things, so keep a sane life balance !

**Comment by steppy** [comment]
I totally agree. Compared to the subsonic apps I worked with so far, Tempo is so much better. It has a modern UI, great usability and is stable. I have been waiting for an app like that for years.

My only problem is the sponsoring as I am unable to donate via buymecoffee.com. Please add a paypal address or another alternative way to support the development.
